crates/bitwarden/src/lib.rs

```diff
@@ -90,4 +90,10 @@ pub use internal::*;
 #[cfg(feature = "secrets")]
 pub mod secrets_manager {
     pub use bitwarden_sm::*;
+
+    pub mod generators {
+        pub use bitwarden_generators::{
+            password, ClientGeneratorExt, PasswordError, PasswordGeneratorRequest,
+        };
+    }
```

Comment:
Any reason to put this under the secrets manager module instead of exposing generators like we do for the `internal` feature?